style.css

```diff
@@ -0,0 +1,441 @@
+  background-color: #fff;
+  color: var(--color-four);
+  border-radius: 8px;
+  border: 1px solid var(--color-one);
+}
+
+.alter {
+  color: #fff;
+  background-color: var(--bg-two);
+}
+
+/* about */
+
+#about-content {
+  display: flex;
+  justify-content: center;
+  background-color: var(--default);
+  border-top-right-radius: 100px;
+}
+
+.content-about-container {
+  display: flex;
+  flex-flow: row nowrap;
+  padding: 142px;
+  justify-content: center;
+}
+
+.block-left {
+  margin-right: 30px;
+  justify-content: center;
+}
+
+.title-about {
+  color: var(--color-three);
+  font-size: 40px;
+  font-weight: bold;
+  margin: 12px 0;
+}
+
+.lang-skills {
+  padding: 24px 0;
+  width: 480px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.img-skill {
+  width: 122px;
+  height: 120px;
+  margin-right: 12px;
+}
+
+.colors-icon {
+  color: var(--color-three);
+}
+
+.icons-foot {
+  padding-left: 10px;
+}
+
+hr {
+  height: 1px;
+  background-color: var(--vector);
+  opacity: 0.2;
+}
+
+.resume-see {
+  width: 180px;
+  margin: 27px auto 0 auto;
+  height: 48px;
+  padding: 12px 16px;
+  font-weight: 500;
+  font-size: 17px;
+  background-color: #fff;
+  color: #6070ff;
+  border-radius: 8px;
+  display: flex;
+  position: absolute;
+}
+
+/* footer */
+
+footer {
+  background-color: var(--default);
+  margin-top: -103px;
+}
+
+.container {
+  background-image: url(image/contact-background.png);
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  display: flex;
+  padding: 105px 24px 8px 24px;
+  margin: 103px auto 0 auto;
+  border-top-left-radius: 100px;
+}
+
+.container section {
+  margin: 0 auto;
+}
+
+.container h1 {
+  font-weight: bold;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 12px;
+}
+
+.container p {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-bottom: 40px;
+}
+
+.container input {
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  width: calc(100% - 16px);
+  padding-left: 16px;
+  height: 48px;
+  font-size: 17px;
+  color: #172b4d;
+}
+
+.container input::placeholder {
+  color: #172b4d;
+}
+
+.form-contacts {
+  width: 447px;
+  margin: 0 auto 127px auto;
+}
+
+.form-contacts textarea {
+  width: calc(100% - 16px);
+  padding: 15px 38px 15px 16px;
+  color: #172b4d;
+  font-size: 15px;
+}
+
+.form-contacts textarea::placeholder {
+  color: #b3bac5;
+}
+
+.get-it-touch {
+  margin: 27px auto 0 auto;
+  width: 140px;
+  height: 48px;
+  padding: 12px 16px;
+  font-weight: 500;
+  font-size: 17px;
+  background-color: #fff;
+  color: #6070ff;
+  border-radius: 8px;
+  display: flex;
+}
+
+.get-it-touch:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button-see-alter:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.get-it-touch:visited {
+  background: #2230d2;
+  color: #fff;
+}
+
+button:hover {
+  cursor: pointer;
+}
```

Comment:
- [ ] Nice implementation of hover interaction on one the buttons. but please make sure you apply it on all the buttons in your portfolio. it is good for the user experience.